staff-user-documentation/README.md

```diff
@@ -0,0 +1,396 @@
+> - It's applied to all privilege purchases across all participating states
+
+#### Transaction Fee Configuration
+
+Compact administrators can configure whether licensees are charged fees to cover payment processor costs
+
+**Licensee Transaction Charges**
+1. Enable/disable transaction fee charging
+2. Set the fee amount to help cover transaction costs
+3. This fee is applied once per privilege purchased
+
+> ⚠️ **Important**: if the compact will not charge transaction fees, leave this section blank.
+
+#### Contact Details for System Notifications
+
+Similar to state administrators, compact administrators must configure notification contacts:
+
+**1. Operations Notification Recipients**
+- Email addresses added to this list will receive notifications related to technical support at the compact level.
+
+**2. Encumbrance Notification Recipients**
+- Email addresses added to this list will receive notifications related to license and privilege encumbrances.
+
+**3. Summary Report Notification Recipients**
+- Email addresses added to this list will receive periodic summary reports generated by the system
+- These reports provide insights into privilege purchasing for all jurisdictions within the compact.
+
+#### Licensee Registration Settings
+
+Compact administrators can enable licensee registration at the compact level. This is the first step to allowing licensee registration. State admins must then also select to enable licensee registration within their individual jurisdictions:
+
+> ⚠️ **Important**: Once enabled, this setting cannot be disabled
+
+---
+
+#### Payment Processing Configuration
+
+Compact administrators can configure Authorize.net payment processing credentials (this must be done after setting up the compact related fees):
+
+**Setup Steps**
+1. Generate API credentials in your Authorize.net account ([view documentation](https://support.authorize.net/knowledgebase/Knowledgearticle/?code=000001271))
+2. Navigate to the "Authorize.net Credentials" section in Compact Settings
+3. Enter your credentials:
+   - **API Login ID**
+   - **Transaction Key**
+4. Click **"Submit"** to enable payment processing
+
+
+> **🔒 Security Considerations:**
+- If credentials are ever compromised, generate new ones immediately in your Authorize.net account
+- Update the credentials in Compact Connect immediately after generating new ones
+- Never share these credentials with others, and do not store them on your device.
+
+### Settings Validation and Confirmation
+
+After making any configuration changes:
+
+1. Review all entered information for accuracy
+2. Click **"Save Changes"** or **"Submit"** as appropriate
+3. Look for confirmation messages indicating successful updates
+4. Some changes may require system processing time before taking effect
+
+⚠️ **Important Notes:**
+- Configuration changes may affect billing and fee collection immediately
+- It is recommended to test any configuration changes in a non-production environment first
```

Comment:
Should we be more explicit and say something like "We recommend that any configuration changes are tested in the Beta environment before being made in Production"?